README.md

```diff
@@ -11,5 +11,5 @@ This repository hosts the source code of [v8.dev, the official website of the V8
 `npm run` shows the full list of supported commands. Highlights:
 
 - `npm run build` builds the site into `dist`.
-- `npm run watch` builds the site into `dist` and watches for changes.
-- `npm start` kicks off a local HTTP server.
+- `npm run serve` kicks off a local HTTP server and watches for changes.
```

Comment:
Please undo these changes. If you're working on CSS/JS changes for v8.dev, you don't want `watch`.

Reply:
Ah, I see `npm run serve` only works when only editing markdown files.